rainforest/branches.go

```diff
@@ -0,0 +1,25 @@
+package rainforest
+
+// Branch is a struct representing a Rainforest Branch
+type Branch struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+}
+
+// CreateBranch creates new branch on RF, requires Branch struct to be prepared to upload using helpers
+func (c *Client) CreateBranch(branch *Branch) error {
```

Comment:
Should we add tests for this? See the tests in `rainforest/runner_test.go` for example

Reply:
I decided not to initially as there isn't really much to test here and I used the `rainforest/tests_test.go` as a reference where this isn't tested explicitly.

After taking a look at the example you suggested, I added a simple test to check the body of the request.
Thanks for the suggestion.